docs/modules/develop/pages/environment.adoc

```diff
@@ -1,5 +1,60 @@
 = Development environment
 
-To start working with the Bulletin Board you will need to install all the dependencies for the server. These are explained in the https://github.com/decidim/decidim-bulletin_board/blob/main/README.md[README file of the web application]. This server will include test data that can be used by Decidim or the elections sandbox to test the different parts of the system.
+**Important: The Bulletin Board depends on the existence of a Decidim installation.**
 
-Also, this application includes Visual Studio Code Dev Containers to simplify the environment setup.
+To start working with the Bulletin Board you will need to install all the dependencies for the server. 
```

Comment:
@beagleknight is working on [a PR to add make for installing and updating the different parts of the project](https://github.com/decidim/decidim-bulletin-board/pull/155), maybe we should add that here?

Reply:
As soon as it's ready, I'll update the documentation again. This was mainly for restructuring and adding some missing parts, like **Adding an authority**.